src/Makefile

```diff
@@ -165,10 +165,12 @@ docker-publish: docker
 
 clean:
 	$(RM) metadatad $(OBJS)
+	(cd ../test && $(MAKE) clean)
```

Comment:
FWIW, each line runs in its own subshell, so there's no need for the parentheses. But that's minor.